hm-post-repeat.php

```diff
@@ -155,9 +159,16 @@ function save_post_repeating_status( $post_id = null, $post_repeat_setting = nul
 
 	if ( 'no' === $post_repeat_setting ) {
 		delete_post_meta( $post_id, 'hm-post-repeat' );
+		return;
 	}
 
-	else {
+	// Backwards compatibility.
```

Comment:
What does this provide backwards compatibility with? We're no longer passing 1 anywhere right?

I think just providing a single backwards compatible layer in `get_repeating_schedule` is enough.